swatch-metrics/src/main/java/com/redhat/swatch/metrics/service/prometheus/PrometheusService.java

```diff
@@ -120,6 +120,14 @@ private String formatErrorMessage(Exception ex) {
    */
   private QuerySummaryResult parseQueryResult(
       File data, Consumer<QueryResultDataResultInner> itemConsumer) {
+    if (log.isDebugEnabled()) {
+      try {
+        log.debug(
+            "Response from prometheus: {}", Files.readString(Path.of(data.getAbsolutePath())));
```

Comment:
I had to manually print the log because the default logger does not print the body when we're dealing with files.